Naveen-gali/Yelp-App
# Events Apis Support

### What does this PR do?

- Add the Yelp Event Apis Support.
- Add the updated response types.
- Update the Pull Request Template

### Related Issues

- closes #21 
- closes #22 
- closes #29 

### How to test

- Clone the Repository.
- Switch to feat/eventsApiIntegration branch.
- Do yarn to install the dependencies.
- Do yarn run build-stage-ios for ios or yarn run build-stage-android for android.
- Now you can see the Home Screen with business response length and a list of 10 events.
- Click on the `featured events in Texas` button.
- You can see the name of the event above the button.
- In the same way by clicking on the event below you can see the name of event coming above the button.
- Note: This is only for illustrative purpose and will be removed.

### Screenshots

|Android ScreenShot           |  Ios ScreenShot|
|:-------------------------:|:-------------------------:
| ![Screenshot_1683105740](https://user-images.githubusercontent.com/122969303/235879196-42b445d4-472b-4d10-bde5-fad4563f4d62.png) | ![Simulator Screen Shot - iPhone 14 - 2023-05-03 at 14 47 10](https://user-images.githubusercontent.com/122969303/235879230-c9028fd9-363b-460b-894b-000225b2bad7.png) |

### Checklist
- [x] I have addressed all the requirements.
- [x] I have self reviewed the PR.

### Additional Notes

- [Link to issue stating deeper in store for persisting](https://github.com/agilgur5/mst-persist/issues/27)
